### PR TITLE
Rebuild

### PR DESCRIPTION
**Pull request by pkgw**
First rebuild in a long time, likely because the FTP download URL has become HTTPS.

Note that upstream is up to version 0.21, but I want to try to get a rebuild in before tackling a new version.

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by pkgw** [comment]
Something's unhappy:

```
(%PREFIX%) %SRC_DIR%>msgfmt.exe -o C:\Users\VSSADM~1\AppData\Local\Temp\tmpy6c769nj\info\recipe\an.gmo C:\Users\VSSADM~1\AppData\Local\Temp\tmpy6c769nj\info\recipe\an.po 
(%PREFIX%) %SRC_DIR%>IF -1073740791 NEQ 0 exit /B 1 
```

That error is 0xC0000409 = `STATUS_STACK_BUFFER_OVERRUN`

**Comment by pkgw** [comment]
It's been a while since I dove into the autotools pain. (Although, reviewing my Windows patches from earlier, I went much deeper last time!) I think this rebuild should now be working, though.

File: recipe/build.sh [hunk 1]
```diff
@@ -44,11 +44,14 @@ if [[ "$target_platform" == win* ]] ; then
...
+    # We need the -MD flag ("link with MSVCRT.lib"); otherwise our executables
+    # can crash with error -1073740791 = 0xC0000409 = STATUS_STACK_BUFFER_OVERRUN
+    #
+    # But -GL messes up Libtool's identification of how the linker works;
     # it parses dumpbin output and: https://stackoverflow.com/a/11850034/3760486
 
-    export CFLAGS=$(echo " $CFLAGS " |sed -e "s, [-/]GL ,,")
-    export CXXFLAGS=$(echo " $CXXFLAGS " |sed -e "s, [-/]GL ,,")
+    export CFLAGS=$(echo "-MD $CFLAGS" |sed -e "s, [-/]GL ,,")
+    export CXXFLAGS=$(echo "-MD $CXXFLAGS" |sed -e "s, [-/]GL ,,")
```
Comment (pkgw) [inline recipe/build.sh:54]: Oops, this adds a problem if -GL is the very last flag, but that doesn't happen in practice and I don't think is worth a rebuild. I'll fix in the 0.21 update.

**Comment by pkgw** [comment]
@conda-forge/gettext If anyone cares to review, I think this is now ready. This PR (or some kind of rebuild) is needed to unblock https://github.com/conda-forge/glib-feedstock/pull/132.

**Comment by ocefpaf** [comment]
> @conda-forge/gettext If anyone cares to review, I think this is now ready. This PR (or some kind of rebuild) is needed to unblock [conda-forge/glib-feedstock#132](https://github.com/conda-forge/glib-feedstock/pull/132).

LGTM!

**Comment by pkgw** [comment]
@ocefpaf The ppc64le Travis build has been in queue all day; is that typical?

**Comment by ocefpaf** [comment]
> @ocefpaf The ppc64le Travis build has been in queue all day; is that typical?

Sadly, yes. Travis-CI is giving us a hard time for a while. If you don't care about ppc64le we can merge as-is and restart the CIs later.

**Comment by pkgw** [comment]
Eh, no rush ...

**Pull request merged**